app/jobs/cache/browser_channels/main.rb

```diff
@@ -28,7 +28,7 @@ def perform
 
   def full_refresh_not_ran_recently?(all_prefixes_run_time:)
```

Comment:
may also consider calling this full_refresh_ran_recently

Reply:
haha I think ruby should implement `not` similar to ! but it's different from python from what I remember: https://stackoverflow.com/questions/27429711/difference-between-not-and-in-ruby